### DISCUSSION TITLE
FIREBIRD Database "alias"

### DISCUSSION THREAD

**Discussion by dpgiorgi** [dbeaver/dbeaver#19965] [Q&A] [answered]
Hi, I can't find a way to define an ALIAS in the configuration of a FIREBIRD database. Sorry if it exists but I can't find it.
Thank you

**Comment by LonwoLonwo** [comment]
Hello @dpgiorgi 

Which kind of alias? Please provide more info.

**Comment by dpgiorgi** [comment]
Lo que necesito es importar datos de una base de datos externa y leí (chatgpt) que lo podría hacer mediante un alias (que se refiere a la otra base de datos) y luego llamar a la tabla que necesito referenciando a ese alias.  Por ejemplo: "select * from alias.articulo"
Estoy buscando solución y por lo que leo debe hacerse con procedimientos almacenados.
Gracias

What I need is to import data from an external database and I read (chatgpt) that I could do it through an alias (referring to the other database) and then call the table I need referencing that alias. For example: "select * from alias.article"
I am looking for a solution and from what I read it must be done with stored procedures.
Thank you

**Reply by mrotteveel** [reply]
Firebird doesn't support cross-database queries (other than by using [`[FOR] EXECUTE STATEMENT ... ON EXTERNAL`](https://firebirdsql.org/file/documentation/html/en/refdocs/fblangref40/firebird-40-language-reference.html#fblangref40-psql-execstmt) in PSQL code), so you cannot use database aliases in `SELECT` (or other) statements.

**Comment by LonwoLonwo** [comment]
This may be what you are looking for. https://firebirdsql.org/rlsnotesh/config-db-alias.html

But I do not see any options to configure it via DBeaver UI.

**Comment by mrotteveel** [comment, marked as answer]
You can only configure this by editing `aliases.conf` (Firebird 2.5 or earlier) or `databases.conf` (Firebird 3.0 or higher). There is no mechanism to add, update or remove aliases through Firebird itself.
